v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/utils/ElasticsearchIO.java

```diff
@@ -1441,6 +1441,11 @@ public void processElement(ProcessContext context) throws Exception {
           }
         }
 
+        long newBatchSizeBytes = currentBatchSizeBytes + document.getBytes(StandardCharsets.UTF_8).length;
```

Comment:
Can you save `document.getBytes(StandardCharsets.UTF_8)` to a variable and reuse below (`currentBatchSizeBytes +=`)?

`String.getBytes` creates a new byte array every time it is called.

Reply:
yes, good idea, let me make that update